### PR TITLE
Add some more dream messages.

### PR DESCRIPTION
**Pull request by Michael1993**
#### Summary
Content "Add new dreams for some mutation categories."

#### Purpose of change

Just wanted to see a bit more variety with mutagen-induced dreams and thought this should be fairly trivial.

#### Describe the solution

Just wrote a couple lines, trying to stay true to the source material.

#### Describe alternatives you've considered

I did not consider any alternatives (aside from not doing it at all, I guess).

#### Testing

I did not test this at all, but the change is very small and straightforward.

#### Additional context

If you guys want, I could try adding some more or rewrite the ones you think wouldn't fit. Also, I think VSCode reformatted the file when I saved it, hope that's not a problem.

### CODE REVIEW

File: data/json/dreams.json [hunk 1]
```diff
@@ -232,14 +234,19 @@
...
     "category": "BEAST",
     "strength": 2
   },
   {
     "type": "dream",
-    "messages": [ "You dream of foraging in the woods… mouth-first?", "Your dream-reflection is rather bearish." ],
+    "messages": [
+      "You dream of foraging in the woods… mouth-first?",
+      "Your dream-reflection is rather bearish.",
+      "You dream of brushing the fur on your back against a tree. Wait, fur?"
```
Comment (I-am-Erk) [inline data/json/dreams.json:248]:
```suggestion
      "You dream of brushing the fur on your back against a tree."
```
There's a good chance the character already has fur, and if they don't, it is more dreamlike this way

File: data/json/dreams.json [hunk 2]
```diff
@@ -271,7 +282,8 @@
     "type": "dream",
     "messages": [
       "You have a dream of working in a hive.",
-      "While dreaming you look into a mirror and see a frighteningly insectoid reflection."
+      "While dreaming you look into a mirror and see a frighteningly insectoid reflection.",
+      "You dream of a feast and excitedly chitter with your pincers... Wait, what?"
```
Comment (I-am-Erk) [inline data/json/dreams.json:286]:
```suggestion
      "You dream of a feast and excitedly chitter with your pincers… wait, what?"
```

File: data/json/dreams.json [hunk 3]
```diff
@@ -280,7 +292,8 @@
     "type": "dream",
     "messages": [
       "You have a confusing dream of growing in a garden.",
-      "You are confused by a plantlike image of yourself in a dream."
+      "You are confused by a plantlike image of yourself in a dream.",
+      "You dream of standing still in the rain, drinking in the falling drops... with your feet?"
```
Comment (I-am-Erk) [inline data/json/dreams.json:296]:
```suggestion
      "You dream of standing still in the rain, drinking in the falling drops… with your feet?"
```

File: data/json/dreams.json [hunk 4]
```diff
@@ -312,7 +325,11 @@
   },
   {
     "type": "dream",
-    "messages": [ "You have a strange dream of spinning webs", "In your dream you see a very spiderlike version of yourself." ],
+    "messages": [
+      "You have a strange dream of spinning webs",
+      "In your dream you see a very spiderlike version of yourself.",
+      "You dream of the buzzing of a fly, that you cannot see. It makes you hungry."
```
Comment (I-am-Erk) [inline data/json/dreams.json:331]:
```suggestion
      "You dream of the buzzing of a fly that you cannot see.  It makes you hungry."
```

File: data/json/dreams.json [hunk 5]
```diff
@@ -329,7 +346,8 @@
     "type": "dream",
     "messages": [
       "You dream of having that success you knew you deserved, back before all this.",
-      "Your dream-self looks competent and in control."
+      "Your dream-self looks competent and in control.",
+      "You dream of a fortress, sheltering humanity from the zombies. Built by you, of course."
```
Comment (I-am-Erk) [inline data/json/dreams.json:350]:
```suggestion
      "You dream of a fortress, sheltering humanity from the zombies.  You're in charge, of course."
```

File: data/json/dreams.json [hunk 6]
```diff
@@ -741,7 +760,7 @@
     "messages": [
       "You dream of the peaceful world you'll give humankind.  Because no one else can.",
       "You crush those who would disagree with the leadership.  Humanity must be united; independence will get you all killed!",
-      "There are underground Faults in the world.  Faults you should remedy."
+      "There are underground Faults in the world. Faults you should remedy."
```
Comment (I-am-Erk) [inline data/json/dreams.json:763]:
```suggestion
      "There are underground Faults in the world.  Faults you should remedy."
```

File: data/json/dreams.json (hunk 4 above)
Comment (Michael1993) [reply data/json/dreams.json:331]: Sorry, why the double-space?

**Comment by Michael1993** [comment]
Sorry, but could you tell me why two-sentence dreams need two spaces between the sentences?

**Comment by I-am-Erk** [comment]
It's part of the style guide, because of fixedwidth fonts we use double spaces after periods for in game strings.

File: data/json/dreams.json [hunk 7]
```diff
@@ -232,14 +234,19 @@
     "type": "dream",
     "messages": [
       "You have a vivid dream of hunting in the woods.",
-      "Whilst dreaming, you see a disturbingly bestial version of yourself."
+      "Whilst dreaming, you see a disturbingly bestial version of yourself.",
+      "You dream of blood and guts. For some reason, it makes you feel weirdly excited."
```
Comment (BrettDong) [inline data/json/dreams.json:238]:
```suggestion
      "You dream of blood and guts.  For some reason, it makes you feel weirdly excited."
```

**Comment by Michael1993** [comment]
Sorry, this is so messy. I'll read a bit more about style guides next time.

**Pull request merged**